packages/bierzo-wallet/src/routes/payment/index.stories.tsx

```diff
@@ -29,20 +35,30 @@ const fullStore = (): DeepPartial<RootState> => {
   };
 };
 
-async function onSubmit(values: object): Promise<void> {
-  const formValues = values as FormValues;
-  console.log('onSubmit');
-  console.log(formValues);
-  return;
+async function onSubmit(_: object): Promise<void> {
+  linkTo(PAYMENT_STORY_PATH, PAYMENT_STORY_CONFIRMATION_PATH)();
 }
 
-storiesOf(WALLET_ROOT, module)
+storiesOf(PAYMENT_STORY_PATH, module)
   .addParameters({ viewport: { defaultViewport: 'responsive' } })
   .add(
-    'Payment page',
+    PAYMENT_STORY_PAYMENT_PATH,
     (): JSX.Element => (
       <DecoratedStorybook storeProps={fullStore()}>
-        <Layout onCancelPayment={action('onCancelPayment')} onSubmit={onSubmit} />
+        <Layout onCancelPayment={linkTo(BALANCE_STORY_PATH, BALANCE_STORY_VIEW_PATH)} onSubmit={onSubmit} />
+      </DecoratedStorybook>
+    ),
+  )
+  .add(
+    PAYMENT_STORY_CONFIRMATION_PATH,
+    (): JSX.Element => (
+      <DecoratedStorybook>
+        <ConfirmPayment
+          transactionId={'transactionID' as TransactionId}
```

Comment:
Could you please use something a little more realistic like an Ethereum transaction ID: `0x2be250c978013e0b3af09916c421511a07fac45bce16cdd891b7001a150cde0e`. Does the screen still work then?